exercises/concept/bird-watcher/.docs/instructions.md

```diff
@@ -0,0 +1,65 @@
+# Instructions
+
+You are an avid bird watcher that keeps track of how many birds have visited your garden.
+
+Usually you use a tally in a notebook to count the birds but you want to better work with your data.
+
+You already digitalized the bird counts per day for the past weeks that you kept in the notebook.
+
+Now you want to determine the total number of birds that you counted, calculate the bird count for a specific week and correct a counting mistake.
+
+## 1. Determine the total number of birds that you counted so far
+
+Let us start analyzing the data by getting a high level view. Find out how many birds you counted in total since you started your logs.
```

Comment:
```suggestion
Let us start analyzing the data by getting a high level view.
Find out how many birds you counted in total since you started your logs.
```